src/app/components/date-range-dropdown/date-range-dropdown.component.scss

```diff
@@ -0,0 +1,11 @@
+:host {
+	display: flex;
+	align-items: center;
+	gap: 8px;
+
+	div {
+		min-width: fit-content;
+		font-size: 18px;
+		margin-bottom: 6px;
```

Comment:
same idea about margin-bottom, is this better handled with line-height?

Reply:
I see that there are code changes, but can you explain? I don't see any implementation involving line-height so I'm left in the dark here

Reply:
In the end I removed margin-bottom: 6px in this change [b869c6e](https://github.com/bitovi/opensource-analytics-dashboard/pull/15/commits/b869c6efa2f30cadc3e337e46be9ab1ea14ed76f) from the dropdown.scss and app.scss and the result was that everything is aligned in center